idealingua-v1/idealingua-v1-core/src/main/scala/izumi/idealingua/model/publishing/manifests/TypeScriptBuildManifest.scala

```diff
@@ -16,20 +16,20 @@ object TypeScriptProjectLayout {
 case class YarnOptions(
                         projectNaming: ProjectNamingRule,
                         scope: String,
-                        dependencies: List[ManifestDependency],
                         devDependencies: List[ManifestDependency],
+                        peerDependencies: List[ManifestDependency],
                       )
 
 object YarnOptions {
   def example: YarnOptions = YarnOptions(
     projectNaming = ProjectNamingRule.example,
-    dependencies = List(
-      ManifestDependency("moment", "^2.20.1"),
-      ManifestDependency("@types/node", "^10.7.1"),
-      ManifestDependency("@types/websocket", "0.0.39"),
-    ),
     devDependencies = List(
-      ManifestDependency("typescript", "3.2.2"),
+      ManifestDependency("typescript", "4.0.1"),
+    ),
+    peerDependencies = List(
+      ManifestDependency("moment", "^2.29.1"),
+      ManifestDependency("@types/node", "^14.11.8"),
+      ManifestDependency("@types/websocket", "1.0.1"),
```

Comment:
Not sure why there are duplicate defines here:
https://github.com/7mind/idealingua-v1/blob/develop/idealingua-v1/idealingua-v1-runtime-rpc-typescript/src/npmjs/package.json

Reply:
@dandriushchenko This defines relates to TS IRT publishing, not for generated sources.
Generated sources use manifests (and TypeScriptBuildManifest.scala contains default manifest if the user does not provide one), but we need to publish internal client libraries so we need defines for them.